### PR TITLE
Add recommendation for licensing to Staging Process

### PR DESCRIPTION
**Pull request by ThisIsMissEm**
Whilst I've recommended the Software and Documents License, there's also additionally licenses for test suites, documents alone and a Intellectual Rights document.

### CODE REVIEW

**Review by tantek — APPROVED** [review]
I support the explicit reference to the W3C S&D License. Thanks Emelia.

**Comment by ThisIsMissEm** [comment]
In the ActivityPub Trust & Safety Taskforce's contributing.md, we have the following:

> Contributions to this repository are intended to become part of Recommendation-track documents governed by the [W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20040205/) and [Software and Document License](https://www.w3.org/Consortium/Legal/copyright-software). To make substantive contributions to specifications, you must either participate in the relevant W3C Community or Working Group or make a non-member patent licensing commitment.

**Comment by evanp** [comment]
There's already a [section on contributions in the CG Charter.](https://swicg.github.io/potential-charters/CGCharter-1727386911.html#contrib) with stronger language:

> Specifications created in the Community Group must use the [W3C Software and Document License](http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document). All other documents produced by the group should use that License where possible.

**Comment by evanp** [comment]
I also think that at least at Stage 0, contributions don't need to be under the CG's IP rules -- in fact, that's the whole point.

I think we should call out when contributions need to be under the CG's contribution rules, and link to those rules directly, rather than reiterating them.

**Comment by ThisIsMissEm** [comment]
I think it's worth pointing out ahead of time that if you want something to get to standards track, then that's the license it'd be under

File: stage-process.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ These stages support the W3C process. If there is ever any question between W3C
 process requirements and how the groups will progress their work, the W3C
 process has precedent.
 
+It is recommended that work undertaken within this staging process should be done
```
Comment (bumblefudge) [inline stage-process.md:25]:
```suggestion
It is recommended that work undertaken intending to be adopted 
by this staging process consider beginning its unofficial work already
```

**Comment by bumblefudge** [comment]
I think this splits the difference-- not recommending it for its own sake, but recommend it be done upfront since it can be more onerous to do post-facto 😄

File: stage-process.md (hunk 1 above)
Comment (dmitrizagidulin) [reply stage-process.md:25]: +1 to this suggestion, merging it in.

**Pull request merged**